docs/components/popovers.md

```diff
@@ -181,7 +181,7 @@ Options can be passed via data attributes or JavaScript. For data attributes, ap
     </tr>
     <tr>
       <td>container</td>
-      <td>string | false</td>
+      <td>string | element | boolean</td>
```

Comment:
Interestingly,  if you actually set `container: true` then the tooltip/popover won't work properly, because of this line:

```
var container = this.config.container === false ? document.body : $(this.config.container);
```

I'm actually going to change these updates back to `false` to make that clear. The important thing for this PR is that `element` is added.